.github/workflows/system-tests-opensource.yml

```diff
@@ -19,18 +19,10 @@ permissions:
   packages: write
 
 on:
-  push:
-    branches:
-      - '.+-os-system-tests'
-
-# disabled until we have a proper schedule definition
-#  schedule:
-#
-#    # * is a special character in YAML so you have to quote this string
-#    # Run the system tests every 3 hours
-#    - cron:  '0 */3 * * *'
-
-  workflow_dispatch:
+   pull_request:
+     branches:
+       - development
```

Comment:
I think it would be safer if this pr will still trigger only on dispatch (and not on pull request).
This way, after the merge we can "play" with it a bit (running several prs in parallel, running failing prs etc).
And then a follow-up pr will enable the trigger on pull-requests.

Reply:
Why can' t we test it against regular PR ?